v2/brigadier/src/groups.ts

```diff
@@ -0,0 +1,45 @@
+import { Job } from "./jobs"
+
+export class Group {
+
+  public static async runAll(jobs: Job[]): Promise<void[]> {
+    const group = new Group(jobs)
+    return group.runAll()
+  }
+
+  public static async runEach(jobs: Job[]): Promise<void> {
+    const group = new Group(jobs)
+    return group.runEach()
+  }
+
+  private jobs: Job[] = []
```

Comment:
Is this necessary if you initialize the jobs in the constructor?

Reply:
Nope. Not necessary. I'll remove that.

Reply:
Fixed in 72fbccf0eddfea14de7e8e0054d9da1a3aba31b1